diagnostic.md

```diff
@@ -40,25 +61,29 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    The serializer lists the attributes associated with a table and it defines what information to obtain from a related table
   ```
 
   ```rb
   class ProfileSerializer < ActiveModel::Serializer
+
+    def profiles
+      object.borrowers.pluck(:title)
+    end
   end
   ```
 
 1.  What would the command be to _scaffold_ out a **join table** for Favorites from
 the above `Movies` and `Profiles`.
 
   ```sh
-    # < Your Response Here >
+    bin/rails generate scaffold favorites reference:movies reference:profiles
   ```
 
 1.  What is `Dependent: Destroy` and where/why would we use it?
 
   ```md
-    # < Your Response Here >
+    Dependent Destroy removes a relationships between tables as stated in a join table.
```

Comment:
`dependent: :destroy` will cause the deletion of an object to also delete all associated objects.  For example, if a `User` model has `has_many :favorites, dependent: :destroy`, then deleting a particular user will also delete all rows in the `favorites` table where `user_id` refers to the `id` of the user being deleted.